src/injected/analyzers/unified-result-sender.ts

```diff
@@ -13,13 +14,15 @@ export class UnifiedResultSender {
         private readonly sendMessage: MessageDelegate,
         private readonly convertScanResultsToUnifiedResults: ConvertScanResultsToUnifiedResultsDelegate,
         private readonly convertScanResultsToUnifiedRules: ConvertScanResultsToUnifiedRulesDelegate,
+        private readonly environmentInfoProvider: EnvironmentInfoProvider,
         private readonly generateUID: UUIDGeneratorType,
     ) {}
 
     public sendResults: PostResolveCallback = (axeResults: AxeAnalyzerResult) => {
         const payload: UnifiedScanCompletedPayload = {
             scanResult: this.convertScanResultsToUnifiedResults(axeResults.originalResult, this.generateUID),
             rules: this.convertScanResultsToUnifiedRules(axeResults.originalResult),
+            toolInfo: this.environmentInfoProvider.getToolData(),
```

Comment:
Sending tool info along with the scan result every time a scan is completed doesn't feel quite right. Tool info should be sent only once, when the 'main window' is initialized.